lib/pharos/scripts/configure-cri-o.sh

```diff
@@ -14,3 +14,8 @@ systemctl enable crio
 # remove unnecessary cni plugins
 rm /etc/cni/net.d/100-crio-bridge.conf /etc/cni/net.d/200-loopback.conf || true
 systemctl start crio
+
+# Install crictl binary
+curl -sSL https://github.com/kubernetes-incubator/cri-tools/releases/download/v1.0.0-beta.0/crictl-v1.0.0-beta.0-linux-amd64.tar.gz | tar xz
```

Comment:
Can we verify this download somehow?

Reply:
We could verify it with a known (== hardcoded) sha or something. I mean verify it manually  first and grab a sha of it and verify against that.

Other option is to package it in our own bintray repo as deb package.